pkg/controller/operators/catalog/operator_test.go

```diff
@@ -182,16 +182,20 @@ func TestSyncInstallPlanUnhappy(t *testing.T) {
 		{
 			// This checks that an installplan is marked as failed when no operatorgroup is present
 			testName:      "HasSteps/NoOperatorGroup",
-			err:           nil,
-			expectedPhase: v1alpha1.InstallPlanPhaseFailed,
-			in:            ipWithSteps,
+			err:           fmt.Errorf("attenuated service account query failed - no operator group found that is managing this namespace"),
+			expectedPhase: v1alpha1.InstallPlanPhaseInstalling,
+			expectedCondition: &v1alpha1.InstallPlanCondition{Type: v1alpha1.InstallPlanInstalled, Status: corev1.ConditionFalse, Reason: v1alpha1.InstallPlanReasonInstallCheckFailed,
+				Message: "attenuated service account query failed - no operator group found that is managing this namespace"},
```

Comment:
Will users have any idea what an "attenuated service account query" is? As far as I know, that language is only used internally.

Reply:
That's a good point. Changing this to `no operator group found that is managing this namespace` only.